jekyll/_cci2/server/installation/hardening-your-cluster.adoc

```diff
@@ -242,6 +246,101 @@ Similar to Nomad clients, there is no need for external VMs to communicate with
 === Egress
 You will only need the egress rules for internet access and SSH for your VCS.
 
+[#notes-on-aws-networkingl]
+=== Notes on AWS Networking with the VM Service
+When using the EC2 module for the VM service, there is a assignPublicIP option available in the values.yaml file.
```

Comment:
Only a minor working suggestion, instead of module, could we use provider? If that's not more appropriate then we can leave as is!

```suggestion
When using the EC2 provider for the VM service, there is a assignPublicIP option available in the values.yaml file.
```